tracer/src/Datadog.Trace/Agent/MessagePack/SpanMessagePackFormatter.cs

```diff
@@ -29,25 +37,36 @@ internal class SpanMessagePackFormatter : IMessagePackFormatter<TraceChunkModel>
         private readonly byte[] _parentIdBytes = StringEncoding.UTF8.GetBytes("parent_id");
         private readonly byte[] _errorBytes = StringEncoding.UTF8.GetBytes("error");
 
-        // name of string tag dictionary
+        // string tags
         private readonly byte[] _metaBytes = StringEncoding.UTF8.GetBytes("meta");
-
-        private readonly byte[] _languageNameBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.Language);
+        private readonly byte[] _languageNameBytes = StringEncoding.UTF8.GetBytes(Tags.Language);
         private readonly byte[] _languageValueBytes = StringEncoding.UTF8.GetBytes(TracerConstants.Language);
-        private readonly byte[] _runtimeIdNameBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.RuntimeId);
+        private readonly byte[] _runtimeIdNameBytes = StringEncoding.UTF8.GetBytes(Tags.RuntimeId);
         private readonly byte[] _runtimeIdValueBytes = StringEncoding.UTF8.GetBytes(Tracer.RuntimeId);
-        private readonly byte[] _originNameBytes = StringEncoding.UTF8.GetBytes(Trace.Tags.Origin);
+        private readonly byte[] _originNameBytes = StringEncoding.UTF8.GetBytes(Tags.Origin);
 
-        // name of numeric tag dictionary
+        // numeric tags
         private readonly byte[] _metricsBytes = StringEncoding.UTF8.GetBytes("metrics");
+        private readonly byte[] _samplingPriorityNameBytes = StringEncoding.UTF8.GetBytes(Metrics.SamplingPriority);
+        private readonly byte[][] _samplingPriorityValueBytes;
```

Comment:
Note to reviewers: `_samplingPriorityNameBytes` and `_samplingPriorityValueBytes` are new. The other changes above is just comments and code cleanup/reformatting.